plugins/templates-maven-plugin/src/main/java/com/google/cloud/teleport/plugin/maven/TemplatesReleaseMojo.java

```diff
@@ -70,18 +70,26 @@ public class TemplatesReleaseMojo extends TemplatesBaseMojo {
   @Parameter(defaultValue = "${gcpTempLocation}", readonly = true, required = false)
   protected String gcpTempLocation;
 
-  @Parameter(
-      name = "baseContainerImage",
-      defaultValue =
-          "gcr.io/dataflow-templates-base/java11-template-launcher-base-distroless:latest",
-      required = false)
+  @Parameter(defaultValue = "${baseContainerImage}", readonly = true, required = false)
```

Comment:
I don't follow this piece - why are we changing the default value here?

Reply:
(same question applies to defaults across the board)

Reply:
Other plugins (i.e. Run and Release) call this plugin and can set those values in theory (though this never happens in practice)

Reply:
Existing example of a parameter not being set and therefore being set to a default value: https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/plugins/templates-maven-plugin/src/main/java/com/google/cloud/teleport/plugin/maven/TemplatesStageMojo.java#L272-L276

Reply:
Right - my question was why we're changing the existing defaults though